app/templates/components/footer-main.hbs

```diff
@@ -102,6 +102,7 @@
       </div>
     </div>
     <div class="ui inverted section divider"></div>
+    <br>
```

Comment:
Don't do it like this

Reply:
Can you little guide ma what is the purpose of the issue I am little confused

Reply:
we need to add/expand the space after the last line, use inline styling.

Reply:
i updated the code